nest-forms-backend/src/utils/guards/thrower-error.guard.ts

```diff
@@ -69,6 +69,22 @@ export default class ThrowerErrorGuard {
     )
   }
 
+  ServiceUnavailableException(
+    errorEnum: string,
+    message: string,
+    console?: string,
+    object?: object,
+  ): HttpException {
+    return this.HttpException(
+      HttpStatus.SERVICE_UNAVAILABLE,
```

Comment:
I do not think this is the correct http code for what is happening in this case https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503

rather I think this is a 400 - bad request